### ISSUE TITLE
PAYDAY 2 Proton version freezes when Steam Overlay is opened with MangoHud enabled

### ISSUE THREAD

**Issue by falxie** [flightlessmango/MangoHud#821]
Running the latest master (https://github.com/flightlessmango/MangoHud/commit/021df9b9324e2624fa815173764ff5d164f66e33) and GE-Proton7-29, and `mangohud %command%`, if the Steam Overlay is opened the game freezes with audio playing in the background indicating that it hasn't crashed, perhaps the Steam Overlay?

**Comment by falxie** [comment]
https://github.com/ValveSoftware/steam-for-linux/issues/8729

**Comment by falxie** [comment]
So strangely I'm unable to get the Steam Overlay to work in PAYDAY 2 with or without MangoHud enabled. Previously I only had these issues with it enabled.

**Comment by falxie** [comment]
I've done a bit more testing and have found that I'm able to replicate the behavior with Proton 7.0. That being that with MangoHud enabled the Steam Overlay freezes upon opening and is fine without. I wonder if it's a DXVK issue in some way, but I didn't have these issues with GTA IV.

**Comment by falxie** [comment]
I was finally able to replicate my original problems which is twofold.

I was using [proton-ge-custom-bin](https://aur.archlinux.org/packages/proton-ge-custom-bin) when I made this issue, but using protonup-qt to install GE-Proton7-29 makes it behave similarly to Proton Experimental or Proton 7.0. The only major difference I can tell between the two install methods is the install location of `/usr/share/steam/compatibilitytools.d` vs `~/.local/share/Steam/compatibilitytools.d`, but I'll just stick to using protonup-qt despite the lack of autoupdate.

Regardless of Proton version, `MANGOHUD=1 %command%` works while `mangohud %command%` doesn't, but sometimes the former will not work without waiting a few minutes after force closing the game after the Steam Overlay crashes, I've noticed there are some lingering gameoverlayui processes. I've also tried using the mangohud executable in `build/src` after running `ninja -C build` with the same results.

There's no difference between using Steam using their runtime vs native either.
